client/blocks/login/index.jsx

```diff
@@ -575,11 +576,16 @@ class Login extends Component {
 				} );
 
 				if ( isGravPoweredLoginPage ) {
+					const isFromGravatar3rdPartyApp =
+						isGravatarOAuth2Client( oauth2Client ) && currentQuery?.gravatar_from === '3rd-party';
+
 					postHeader = (
 						<p className="login__header-subtitle">
-							{ translate(
-								'If you prefer logging in with a password, or a social media account, choose below:'
-							) }
+							{ isFromGravatar3rdPartyApp
+								? translate( 'Please log in with your email and password.' )
```

Comment:
I think this message is wrong. There is no password, right? Maybe this should be "Please log in with your email.". And that's all?

Reply:
Nevermind, I just saw that this is only for email/password flow. All good!